jjb/egeria/egeria-docker.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- project:
+    name: egeria-configure
+    project-name: egeria-configure
+    project: egeria
+    mvn-settings: egeria-settings
+    archive-artifacts: ''
+    build-node: centos7-docker-4c-4g
+    cron: 'H 11 * * *'
+    mvn-snapshot-id: logs
+    container-public-registry: ''
+    container-snapshot-registry: ''
+    stream:
+      - 'master':
+          branch: master
+    jobs:
+      - github-maven-docker-merge:
+          mvn-goals: 'clean install -Ddocker -Ddocker.repo=odpiadmin -Ddockerfile.useMavenSettingsForAuth=true -f open-metadata-resources/open-metadata-deployment/docker/configure'
```

Comment:
mvn-goals should only be the actual goals, everything else is mvn-params.
Is there a reason why the team is using clean install instead of clean deploy?

Reply:
mvn deploy will deploy images to nexus. In this project, the phase is defined as install to deploy images to docker hub

Reply:
Please make sure you change this variable. 
mvn-goals should only be the actual goals, everything else is mvn-params.

Reply:
@jwagantall, Incorporated review comments.. Now separated mvn-goals and mvn-params

Also changed the job from merge job to verify job. As we are not using any nexus in our docker, verify job is sufficient for our job.